src/containers/navbar-left/navbar-left.js

```diff
@@ -39,7 +39,7 @@ const NavbarLeft = () => {
   return (
     <Toolbar>
       <Typography variant='h6'>
-        <Link to='/' className={styles.logo}>
+        <Link id='logo' to='/' className={styles.logo}>
```

Comment:
Why do we need an id here?

Reply:
I wanted to use it for cypress test. it is not needed any more. fixed